apps/prairielearn/src/tests/utils/enrollments.ts

```diff
@@ -0,0 +1,59 @@
+import { assert } from 'chai';
+import fetch from 'node-fetch';
+import { z } from 'zod';
+import { queryRow } from '@prairielearn/postgres';
+
+// Must be imported so that `config.serverPort` is set.
```

Comment:
This is kinda gross, but 🤷 

I looked into using Mocha hooks to apply these changes to each worker process so that we wouldn't have to rely on a side effect of importing this module, but couldn't figure out something that worked.